etc/elasticsearch.yml

```diff
@@ -0,0 +1,130 @@
+# etc/elasticsearch.yml
+AWSTemplateFormatVersion: "2010-09-09"
+Description: >
+  Creates an Amazon Elasticsearch Service (Amazon ES) domain, with a VPC
+  endpoint, automated snapshots, and an optional EBS volume.
+Parameters:
+  ProjectTag:
+    Type: String
+  EnvironmentTag:
+    Type: String
+    AllowedValues:
+      - Staging
+      - Production
+  EbsEnabled:
+    Type: String
+    AllowedValues:
+      - true
+      - false
+  EbsVolumeType:
+    Type: String
+    AllowedValues:
+      - standard
+      - gp2
+  EbsVolumeSize:
```

Comment:
Maybe worth noting that this isn't a straightforward number?  The web console has some notes that you should...

> multiply the EBS storage size per node by the number of data nodes in your cluster to calculate the total storage available to your cluster